ecl/eclccserver/eclccserver.cpp

```diff
@@ -392,6 +493,11 @@ class EclccCompileThread : implements IPooledThread, implements IErrorReporter,
             workunit->getDebugValue(debugStr.str(), valueStr);
             processOption(debugStr.str(), valueStr.str(), eclccCmd, eclccProgName, *pipe, true);
         }
+        bool compileCppSeparately = globals->getPropBool("@compileCppSeparately", false);
```

Comment:
I think we should consider defaulting this on.
May also want the option to override it for a particular workunit.